src/neuroglancer/datasource/graphene/backend.ts

```diff
@@ -127,6 +128,82 @@ async function decodeDracoFragmentChunk(
   }
 }
 
+interface ShardInfo {
+  shardUrl: string;
+  offset: Uint64;
+}
+
+interface GrapheneMultiscaleManifestChunk extends MultiscaleManifestChunk {
+  fragmentIds: FragmentId[]|null;
+  shardInfo?: ShardInfo;
+}
+
+function decodeMultiscaleManifestChunk(chunk: GrapheneMultiscaleManifestChunk, response: any) {
+  verifyObject(response);
+  chunk.manifest = {
+    chunkShape: vec3.clone(response.chunkShape),
+    chunkGridSpatialOrigin: vec3.create(),
+    lodScales: new Float32Array(response.lodScales),
+    octree: new Uint32Array(response.octree),
+    vertexOffsets: new Float32Array(response.lodScales.length * 3),
+    clipLowerBound: vec3.create(),
```

Comment:
This is why the meshes aren't displaying properly --- the mesh rendering code calculates bounds for each octree node by calculating the bounds of the node according to the grid, and then restricting them by these overall clip bounds.  If the resultant bounding box does not intersect the view frustum, then the octree node won't be rendered.  With these values for the clip bounds, all of the calculated bounds end up being (0, 0, 0)---(0,0,0), which both defeats the normal logic of selectively rendering only visible chunks, and also means if the view frustum does not contain (0, 0, 0) then nothing will be rendered.

Reply:
If you don't have the real bounds available, but you can just calculate the bounds that include all of the fragments present.

Reply:
Thanks! I have pushed changes and updated the chunkedgraph deployment to return clip bounds.

Realized that zoom-out disappearance is being caused by resolution, changing it fixes that. But this means higher resolution meshes are loaded earlier than expected.

Currently chunkedgraph is passing bounding box of the dataset for clip lower and upper bounds. Should that fix the above issue? It doesn't look like it does though, I'm not familiar with webGL.